src/screens/profile/MyAccommodations/MyAccommodations.tsx

```diff
@@ -72,7 +72,7 @@ const MyAccommodations = ({ navigation }: Props) => {
     dispatch(accommodationActions.clearError());
     dispatch(myAccommodationsListActions.clearError());
     fetchMyAccommodations();
-  }, [dispatch]);
+  }, [dispatch, fetchMyAccommodations]);
```

Comment:
isn't it causing infinite fetching when you add fetchMyAccommodations to dependency? better check on more time

Reply:
no, I checked, there is noting that cause infinite fetching, also function is wrapped with useCallback, also this removes warning of useEffect missing dependency